app/src/main/java/com/example/qrscore/Account.java

```diff
@@ -0,0 +1,35 @@
+package com.example.qrscore;
+
+import java.util.List;
+
+// TODO: As a player, I want to add new QR codes to my account.
+// TODO: As a player, I want to see what QR codes I have added to my account.
+// TODO: As a player, I want to remove QR codes from my account.
+// TODO: As a player, I want to see my highest and lowest scoring QR codes.
+public class Account {
+    private List<QRCode> qrCodes;
+    private String device;
+    private Profile profile;
+    // private Stats stats;
+    // private Permissions permissions;
+
+    public List<QRCode> getQRCodes() {
+        return qrCodes;
+    }
+
+    public void addQRCode(QRCode toAdd) {
```

Comment:
Instead of toAdd as the param name, I feel like we should just leave it as qrCode for consistency. As we know this method is used to add a qrCode.